### ISSUE TITLE
Author unable to edit Kata or open translations

### ISSUE THREAD

**Issue by rollietikes** [Codewars/codewars.com#770]
I have been receiving an error since last night while trying to update this Kata in any language or open translations. At this point there is no way to edit the Kata at all.

https://www.codewars.com/kata/help-kiyo-kiyo-solve-her-problems-lcm-fun/ruby

If I try to open any translations or create a new one I get a server 500 error. 

If I am inside of the Kata translator and try to edit the Kata under any language I receive "Final Solution is invalid error".

I finally had to delete my Java language version last night. This does not seem to be an issue I can fix on my side please advise.
![screenshot from 2017-01-10 09-31-24](https://cloud.githubusercontent.com/assets/7141278/21817191/e179b9fa-d717-11e6-8b43-4b0127141145.png)

![screenshot from 2017-01-10 09-29-47](https://cloud.githubusercontent.com/assets/7141278/21817203/e819b8c8-d717-11e6-8889-53d032374954.png)

![screenshot from 2017-01-10 09-25-34](https://cloud.githubusercontent.com/assets/7141278/21817209/ee4efb04-d717-11e6-9a33-084b242e7e93.png)

**Comment by jhoffner** [comment]
I was just able to approve a translation, looks like translations can be created as well. I however could reproduce the kata editor "final solution is invalid" issue.

**Comment by suic86** [comment]
Hi,

I can confirm this issue too. I got the same while editing [my own kata](https://www.codewars.com/kata/character-with-longest-repetition/discuss#5874e37f3793a300830001bf) and also [raulbc777's kata](https://www.codewars.com/kata/begin-your-day-with-a-challenge-but-an-easy-one/forks/haskell).

Regards,

suic

**Comment by rollietikes** [comment]
BTW- I am unable to go into the translator for this Kata even pressing the "+" button  results in a 500 error.

**Comment by jhoffner** [comment]
Should be fixed now

**Comment by jhoffner** [comment]
Well, the "Final solution is invalid" issue should be fixed at least. The translation issue seems to be different, though I wasn't able to reproduce. I approved the haskell translation so maybe @suic86 tried to edit it after I approved it? (you can't edit them after they are approved)

**Comment by rollietikes** [comment]
I can edit the Kata now but I cannot enter a new translation without a 500 error. If there is no way to fix this I can just create it in the Kata editor.

![screenshot from 2017-01-10 19-44-59](https://cloud.githubusercontent.com/assets/7141278/21834888/7f96dc44-d76d-11e6-9fba-da5fea4b4526.png)

**Comment by rollietikes** [comment]
So I can edit Ruby, JS, and Python fine but I am trying to add Java in the Kata editor and get the same error. I had to delete this Kata last night and am trying to create it again when I publish I get the same error.
Any chance you can do this fix for Java as well? 
![screenshot from 2017-01-10 20-23-07](https://cloud.githubusercontent.com/assets/7141278/21835583/f703ecf4-d772-11e6-8218-4279275a15f4.png)

Example Fixture is invalid. (java) The example test cases should pass when ran against the complete solution

**Comment by rollietikes** [comment]
All issues seem to be resolved at this point... Thanks Jake!

**Comment by suic86** [comment]
@jhoffner and @rollietikes this still doesn't work for Haskell for [this kata](https://www.codewars.com/kata/586d6cefbcc21eed7a001155/solutions/haskell). To update this kata I've needed to republish this 3 times in a row. First time I got the red rectangle with the message. Second time I got the rectangle without the message. Third time it published the kata but not the solution. Hower my solutions aren't published :(

Thanks,

suic
